plugins/operators/mattermost_operator.py

```diff
@@ -0,0 +1,60 @@
+from airflow.plugins_manager import AirflowPlugin
+# airflow.providers.http.operators.http -> airflow 2
+from airflow.operators.http_operator import SimpleHttpOperator
+from hooks.mattermost_webhook_hook import *
+
+
+class MattermostOperator(SimpleHttpOperator):
+
+    """
+    Operator that allows sending messages to Mattermost. For more details on the parameters, please check
+    `Incoming webhooks <https://developers.mattermost.com/integrate/webhooks/incoming/#parameters>_` documentation.
+    :param http_conn_id: :ref:`http connection<howto/connection:http>` that has the base
```

Comment:
```suggestion
    :param mattermost_conn_id: :ref:`http connection<howto/connection:http>` that has the base
```

Small inconsistency between docs and code.